source/templates/fhs-simple-split/template.htm

```diff
@@ -0,0 +1,128 @@
+<link rel="icon" type="image/x-icon" href="<?php echo $pth['folder']['template']?>favicon.ico">
+<link rel="icon" type="image/gif" href="<?php echo $pth['folder']['template']?>favicon.gif">
+<link rel="icon" type="image/png" href="<?php echo $pth['folder']['template']?>favicon.png">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon.png">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-57x57.png" sizes="57x57">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-60x60.png" sizes="60x60">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-72x72.png" sizes="72x72">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-76x76.png" sizes="76x76">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-114x114.png" sizes="114x114">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-120x120.png" sizes="120x120">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-128x128.png" sizes="128x128">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-144x144.png" sizes="144x144">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-152x152.png" sizes="152x152">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-180x180.png" sizes="180x180">
+<link rel="apple-touch-icon" href="<?php echo $pth['folder']['template']?>apple-touch-icon-precomposed.png">
+<link rel="icon" type="image/png" href="<?php echo $pth['folder']['template']?>favicon-16x16.png" sizes="16x16">
+<link rel="icon" type="image/png" href="<?php echo $pth['folder']['template']?>favicon-32x32.png" sizes="32x32">
+<link rel="icon" type="image/png" href="<?php echo $pth['folder']['template']?>favicon-96x96.png" sizes="96x96">
+<link rel="icon" type="image/png" href="<?php echo $pth['folder']['template']?>favicon-160x160.png" sizes="160x160">
+<link rel="icon" type="image/png" href="<?php echo $pth['folder']['template']?>favicon-192x192.png" sizes="192x192">
+<link rel="icon" type="image/png" href="<?php echo $pth['folder']['template']?>favicon-196x196.png" sizes="196x196">
+</head>
+
+<body<?php echo onload();?>>
+<div class="row-full-width" id="header">
+<div class="row" id="image">
+<div class="bild">
+<div id="logo"><a href="./" class="logo">&nbsp;</a></div>
+<div id="logoTxt"><a href="./">&nbsp;CMSimple_XH<span class="red smaller">&lt;split&gt;</span> &ndash; &shy;Das&nbsp;simple&nbsp;CMS</a></div>
+</div>
+</div>
+<div class="burger"><i class="fa fa-bars fa-2x"></i></div>
+</div>
+<nav class="nav_horizontal">
+<div class="nav_row"> <?php echo toc();?> </div>
+</nav>
+<div class="row-full-width" id="wrapper">
+<div class="row content fullWidht fullWidhtNoLineTop">
```

Comment:
> fullWidht und fullWidhtNoLineTop sehen nach Tippfehler aus (müsste doch eigentlich width lauten).
Nein, muss es nicht, weil die Klasse genau so auch in der [stylesheet.css](https://github.com/TN03/XH_split/pull/25/files#diff-894397bce18e01920faebdcb7dfbd402R374) definiert ist.